components/learn/modules/ROOT/pages/dip-staking.adoc

```diff
@@ -2,86 +2,120 @@
 
-*DETAILS* +
+==== DETAILS
 On the table's far right, you can click '`DETAILS.`'
 
+== Manage your DIP staking
+
+In the detail view, you can stake DIP token or manage your staked DIP token.
+
 image::_images/dip-3-details.png[]
 
 Here you will find detailed information about the risk bundle. The general information and personal information like '`My Supported Capital.`' +
 The buttons where you can act are highlighted in blue. +
-*STAKE* +
+
+==== STAKE
 You can start staking or stake more DIP token in this risk bundle.  +
-*UNSTAKE* +
+
+==== UNSTAKE
 The '`UNSTAKE`' button in the screenshot is not active. You cannot unstake your DIP token in the example screen because the risk bundle is in the state '`Active.`' Unstaking is only available for expired risk bundles or in state '`Closed`' or '`Burned.`' +
 Please note: Your DIP token are not at risk. +
-*CLAIM REWARDS* +
-You can claim your rewards at any time during the lifetime of a risk bundle, but at the latest when you unstake the total amount of your staked DIP tokens for this bundle.
 
-=== Stakes
+==== CLAIM REWARDS
+You can claim your rewards at any time during the lifetime of a risk bundle, but at the latest when you unstake the total amount of your staked DIP tokens for this bundle. +
+
+==== RESTAKE
+
+Restaking is only possible in risk bundles that are already closed or expired. You can only restake the total amount of DIP token you staked in this risk bundle. You can only restake in a risk bundle in which you have not yet staked any DIP token.
+
+== Tab '`Stakes`'
 
 This area is identical to the '`DIP Staking`' menu item. You can see all existing risk bundles.
 
-=== Stake DIP
-test
+== Tab '`Stake DIP`'
+
 image::_images/dip-4-stakingdip.png[]
 
 In this area, you will only see the risk bundles in which you can stake DIP token. By clicking '`SELECT`' on the right, compare the conditions and terms and choose a risk bundle. You see all details of the risk bundle.
-the minimum volume is 5,000 DIP
+The minimum volume is 5,000 DIP.
+
+==== Gasless staking
+
+By checking the box, '`I would like Etherisc to submit the transaction and pay fees on my behalf,`' you can set Etherisc to pay the fees. 
+
+IMPORTANT: Etherisc actually limits the fees to 35 Gwei. You can see the current limit by hovering over the info button. The text that appears contains the current limit. Etherisc does not guarantee that the transaction will be executed immediately.
+
+You will see this hint if the transaction still needs to be performed.
+
+image::_images/dip-9-pending-fee.PNG[]
+
+You can only check this box once at the initial staking. We take over the fees for the staking in a risk bundle only once per wallet. If you want to stake more DIP tokens in another risk bundle, we will take over these costs again at the initial staking. +
```

Comment:
> We take over the fees for the staking in a risk bundle only once per wallet. 

This is a bit misleading as it might lead people to believe we cover only one gasless per wallet, but actually we cover one gasless per wallet and bundle. Maybe better

`We cover the fees for the initial stake in a risk bundle that the current wallet has not yet staked in.`